app/pages/project/classify.cjsx

```diff
@@ -321,6 +326,13 @@ module.exports = React.createClass
       apiClient.type('user_groups').get(props.location.query.group).then (group) =>
         @setState({ validUserGroup: group and group.links.users.includes(props.user.id) })
```

Comment:
if there is a group query param in the url and a signed in user, but the user isn't included in the group, do we want to clear the group query param from the url? could conditionally call `clearUserGroupForClassification` within `validateUserGroup` if user not included in group.